GEOSldas_App/ldas_setup

```diff
@@ -250,21 +250,26 @@ class LDASsetup:
                exit("Error. LANDASSIM_DT is inconsistent with ADAS analysis window.\n")
            self.rqdExeInp['LANDASSIM_T0']             = "013000"                                 # HHMMSS
            jsgmt1                                     = "00000000"
-           jsgmt2                                     = hours_to_hhmmss(int(self.varwindow)/60)       # convert minutes to HHMMSS
+           jsgmt2                                     = hours_to_hhmmss(int(self.varwindow)/60)  # convert minutes to HHMMSS
            self.rqdExeInp['JOB_SGMT']                 = f"{jsgmt1} {jsgmt2}"
            self.rqdExeInp['NUM_SGMT']                 = 1 
            self.rqdExeInp['FORCE_DTSTEP']             = 3600
 
            # determine END_DATE = BEG_DATE + TIME_STEP_OF_ADAS_CYCLE
-           _beg_date = datetime.strptime( self.rqdExeInp['BEG_DATE'], "%Y%m%d %H%M%S")
-           _hours    = int(self.rqdExeInp['JOB_SGMT'][ 9:11])
-           _end_date = _beg_date + timedelta(hours=int(self.varwindow)/60) 
-
+           _beg_date                                  = datetime.strptime( self.rqdExeInp['BEG_DATE'], "%Y%m%d %H%M%S")
+           _hours                                     = int(self.rqdExeInp['JOB_SGMT'][ 9:11])
+           _end_date                                  = _beg_date + timedelta(hours=int(self.varwindow)/60) 
            self.rqdExeInp['END_DATE']                 = _end_date.strftime("%Y%m%d %H%M%S") 
-           self.rqdExeInp['NML_INPUT_PATH']           = self.LDAS_nml_path
-           self.rqdExeInp['MWRTM_PATH']               = self.mwrtm_path   
+
+           # find nens  from  ldas ICS
+           rstdir=self.rqdExeInp['RESTART_PATH']+'/'+self.rqdExeInp['RESTART_ID']+'/output/'+ self.rqdExeInp['RESTART_DOMAIN'] 
+           enslist = glob.glob(os.path.join(rstdir, "rs", "ens????"))  
+           self.rqdExeInp['NUM_LDAS_ENSEMBLE']         = len(enslist) 
+           # if fvsetup send atm_nens via commandline, use that
+           if self.ladas_cpl == 2 :
+              self.rqdExeInp['NUM_LDAS_ENSEMBLE']      = self.nens 
```

Comment:
I'm a bit confused why we do this here and not in fvsetup.  In offline mode, the LDAS can restart from a single member and "cold-start" the ensemble.  This modification may be conflicting with the ens cold-start functionality.  If we count the number of members in fvsetup, we don't need to change anything in ldas_setup

Reply:
@gmao-rreichle  we find nens only in case of ldas_cpl > 0 , it will not conflict with any other general ldas cases ( such as cold-start) .    I implemented finding atmens nens in fvsetup and pass it to ldas_setup  in command line ( only in the ldas_setup 4atmens call), tested but not yet pushed.  Alternatively we can make fvsetup find all nens ( ldas det,  ldas_4ens,  atmens), and pass nens to ldas_setup in command line.

Reply:
Ok, thanks.  I think it would be better to have all the counting done in one place (fvsetup).

Reply:
it is changed in fvsetup to get all nens (adas PR#352 commit 54dacbc) . ldas_setup get nens via command line.